# AuthBugDemo
Demo to reproduce a `bug` that occurs when `context.Fail` is called within the authorization handler.

Reported [here](https://github.com/dotnet/aspnetcore/issues/55673).
